config/config.exs

```diff
@@ -26,6 +26,11 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+config :flat_search, FlatSearch.Scheduler,
+  jobs: [
+    {"*/5 * * * *", fn -> FlatSearch.OlxScraper.run() end}
```

Comment:
writing this anonymous function as `{FlatSearch.OlxScraper, :run, []}` would be more by the book in this case 😛